course/viewbase.py

```diff
@@ -57,6 +58,7 @@ def get_resource_objects(self):
             self.instance = instance
             self.course = self.instance.course
             self.content = CachedContent(self.instance)
+            self.exam_sessions = ExamSession.objects.all()
```

Comment:
This class `CourseInstanceBaseMixin` is related to a single course instance, so why are all ExamSessions in the database retrieved here? Why not only the ExamSessions related to this course instance?

Reply:
```suggestion
            self.exam_sessions = ExamSession.objects.filter(
                course_instance=self.instance)
```

Reply:
Looks like you did not commit this change yet. The suggested change looks fine. Only the exam sessions of this one course instance should be fetched here.

Reply:
Commited now